### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by Wahaj-Ali**
### Highlights
Screenshot 2023-02-20 205410
Screenshot 2023-02-20 205329
Screenshot 2023-02-20 205301
Screenshot 2023-02-20 205250
Screenshot 2023-02-20 205235
Screenshot 2023-02-20 205222
Screenshot 2023-02-20 205206
Screenshot 2023-02-20 205136

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ CREATE TABLE animals(
+    id INT,
```
Comment (ShadyShawkat) [inline schema.sql:4]: - [ ] [OPTIONAL] You can use the following code to set the `id` column as a primary key:
```SQL
id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+ SELECT * FROM animals WHERE name LIKE '%mon%';
+ SELECT name FROM animals WHERE date_of_birth>= '2016-01-01' AND date_of_birth<= '2019-01-01';
```
Comment (ShadyShawkat) [inline queries.sql]: - [x] Please make sure that this query selects any birth date between the years `2016` and `2019` which means you have to select until the end of `2019`, meaning `'2019-12-31'` not `'2019-01-01'`.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+ SELECT name FROM animals WHERE neutered IS TRUE AND escape_attempts<3;
+ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
+ SELECT name,escape_attempts FROM animals WHERE weight_kg<10.5;
```
Comment (ShadyShawkat) [inline queries.sql:7]: - [x] Kindly make sure to select any animal that weighs more than `10.5kg` not less.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Wahaj-Ali, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @Wahaj-Ali, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job addressing the previous comment :100: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**